src/components/common/ClaimItem.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  export let title;
+  export let amount;
+  export let loading = false;
+  export let onClick;
+
+  const onClaimClicked = () => {
+    loading = true;
+    onClick()
+  }
+</script>
+
+{#if loading === false}
+  <div class="container" on:click={() => onClaimClicked()}>
+    <div class="description">
+      <span class="title">{title}</span>
+      <span class="value">${amount}</span>
+    </div>
+    <div class="description">
+      <span class="claim">Claim</span>
+    </div>
+  </div>
+{:else}
+  <div class="container">
+    <div class="description">
+      <span class="value">Loading</span>
+    </div>
+  </div>
```

Comment:
I will suggest this loading element will only replace the button instead of the entire content